### DISCUSSION TITLE
Dynamic dialog animation control

### DISCUSSION THREAD

**Discussion by amrkh97** [primefaces/community#962] [PrimeNG] [answered]
Hi, we are using Dynamic Dialog with these configurations:
```position: 'top-right',
transitionOptions: '150ms cubic-bezier(0, 0, 0.2, 1)'
```

But currently we want to change the transition to be opening from the side, instead of the pop-up mode.

Is there a way to do this currently ?

Also, some questions if this is the right place for it.
For the header, why can't it accept a template ? as of right now, if I add some configurations like: draggable, close, close on escape, maximizable and any other header operation it doesn;t work, is there a way to enable these operation without having to use the default header or a header component ?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You can use transitionOptions property to customize animation, there are a lot of animation tools on the web to generate what you need you can use one of them.
For the header, we've introduced a feature that supports passing components as template pieces. Here is the example containing footer template as a separate component, you can do the same thing for the header too. https://primeng.org/dynamicdialog#example
